src/ScheduleFunctions.cpp

```diff
@@ -1438,19 +1293,27 @@ class InjectGroupRealization : public IRMutator2 {
             const Function &f = iter->second;
             string prefix = pair.func_2 + ".s" + std::to_string(pair.stage_2) + "." + pair.var_name;
             if (visited.find(prefix) == visited.end()) {
-                const Definition &def_2 = (pair.stage_2 == 0) ? f.definition() : f.update(pair.stage_2 - 1);
+                const Definition &def_2 = (pair.stage_2 == 0) ? f.definition() : f.update(
+                        static_cast<int>(pair.stage_2 - 1));
                 collect_all_dependence_helper(skip, prefix, def_2, pair, dependence, visited);
             }
         }
         return dependence;
     }
 
+
     // Replace the bounds of the parent fused loop (i.e. the first one to be
     // realized in the group) with union of the bounds of the fused group.
-    Stmt replace_parent_bound_with_union_bound(const map<string, bool> &skip, const Function &f,
-                                               Stmt produce, const map<string, Expr> &bounds) {
+    Stmt replace_parent_bound_with_union_bound(const map<string, bool> &skip, const Function &f, Stmt produce,
+                                               const map<string, Expr> &bounds) {
         string prefix = f.name() + ".s0";
         const Definition &def = f.definition();
+
+        if(!def.defined())
+        {
```

Comment:
brace goes on end of previous line. Using the clang-format file may help catch these